tests/communication/unit/test_semeio_script.py

```diff
@@ -6,18 +6,27 @@
 
 from semeio.communication import SEMEIOSCRIPT_LOG_FILE, SemeioScript
 
-from unittest.mock import Mock
+from unittest.mock import Mock, PropertyMock
 
+def _ert_mock(config_path="config_path", user_config_file="config_file"):
+    resconfig_mock = Mock()
+    config_path_prop_mock = PropertyMock(return_value=config_path)
+    user_config_prop_mock = PropertyMock(return_value=user_config_file)
+    type(resconfig_mock).config_path = config_path_prop_mock
+    type(resconfig_mock).user_config_file = user_config_prop_mock
```

Comment:
Do you intend to do this:
```suggestion
    resconfig_mock = Mock()
    resconfig_mock.config_path = config_path
    resconfig_mock.user_config_file = user_config_file

    return Mock(
        resConfig=Mock(
            return_value=resconfig_mock
        )
    )
```

Reply:
I was given the impression from the docs that one should use `PropertyMock` instead of modifying the `__set__`'er.

But it didn't look particularly nice when one has to use the `type` in order to set the `PropertyMock` on the `Mock` itself (which again, is from the documentation)..

Reply:
I'll change it though to the proposed example.

Reply:
Depends what you want to do - do you want to assert on the property? Then yes, definitely use `PropertyMock`

Reply:
As in, assert that it has been called? Not doing it for now at least.